<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGrassToolsBase</class>
 <widget class="QgsDockWidget" name="QgsGrassToolsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GRASS Tools</string>
  </property>
  <widget class="QWidget" name="mContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="mMessageLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;No mapset is open. You can open a GRASS mapset from the browser using the mapset item's context menu action &lt;span style=&quot; font-style:italic;&quot;&gt;Open mapset&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="mTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mModulesTreeTab">
       <attribute name="title">
        <string>Modules</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="mViewModeButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../images/images.qrc">
              <normaloff>:/images/themes/default/mIconListView.png</normaloff>:/images/themes/default/mIconListView.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Filter</string>
            </property>
            <property name="margin">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mFilterInput"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="mTreeView">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="mListView">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="mDebugWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="mDebugLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mDebugReloadButton">
             <property name="text">
              <string>Reload tree</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mDebugButton">
             <property name="text">
              <string>Run debug</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mCloseDebugButton">
             <property name="text">
              <string>Close debug</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>mTabWidget</zorder>
   <zorder>mMessageLabel</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDockWidget</class>
   <extends>QDockWidget</extends>
   <header>qgsdockwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
